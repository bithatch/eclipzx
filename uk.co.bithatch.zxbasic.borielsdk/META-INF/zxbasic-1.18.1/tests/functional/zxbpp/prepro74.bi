#include once <pos.bas>

#ifndef __D7_BAS__
#define __D7_BAS__
#include "prepro74.bi"
#endif

glibberish
