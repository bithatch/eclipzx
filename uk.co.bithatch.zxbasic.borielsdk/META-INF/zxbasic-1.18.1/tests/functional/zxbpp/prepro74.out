#line 1 "prepro74.bi"
#line 1 "/zxbasic/src/lib/arch/zx48k/stdlib/pos.bas"













#line 15 "/zxbasic/src/lib/arch/zx48k/stdlib/pos.bas"

#pragma push(case_insensitive)
#pragma case_insensitive = true









function FASTCALL pos as ubyte
    asm
    push namespace core
    PROC

    call __LOAD_S_POSN
    ld a, e

    ENDP
    pop namespace
    end asm
end function

#pragma pop(case_insensitive)
#require "sposn.asm"


#line 45 "/zxbasic/src/lib/arch/zx48k/stdlib/pos.bas"
#line 2 "prepro74.bi"



#line 1 "prepro74.bi"


#line 7 "prepro74.bi"

glibberish
#line 6 "prepro74.bi"
#line 7 "prepro74.bi"

glibberish
